admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -24,153 +24,166 @@
  */
 
 export default {
-  orderPaymentDetailsBtn: '.js-payment-details-btn',
-  orderPaymentFormAmountInput: '#order_payment_amount',
-  orderPaymentInvoiceSelect: '#order_payment_id_invoice',
-  viewOrderPaymentsBlock: '#view_order_payments_block',
-  privateNoteToggleBtn: '.js-private-note-toggle-btn',
-  privateNoteBlock: '.js-private-note-block',
-  privateNoteInput: '#private_note_note',
-  privateNoteSubmitBtn: '.js-private-note-btn',
-  addCartRuleModal: '#addOrderDiscountModal',
-  addCartRuleApplyOnAllInvoicesCheckbox: '#add_order_cart_rule_apply_on_all_invoices',
-  addCartRuleInvoiceIdSelect: '#add_order_cart_rule_invoice_id',
-  addCartRuleTypeSelect: '#add_order_cart_rule_type',
-  addCartRuleValueInput: '#add_order_cart_rule_value',
-  addCartRuleValueUnit: '#add_order_cart_rule_value_unit',
-  cartRuleHelpText: '.js-cart-rule-value-help',
-  updateOrderStatusActionBtn: '#update_order_status_action_btn',
-  updateOrderStatusActionInput: '#update_order_status_action_input',
-  updateOrderStatusActionForm: '#update_order_status_action_form',
-  showOrderShippingUpdateModalBtn: '.js-update-shipping-btn',
-  updateOrderShippingTrackingNumberInput: '#update_order_shipping_tracking_number',
-  updateOrderShippingCurrentOrderCarrierIdInput: '#update_order_shipping_current_order_carrier_id',
-  updateCustomerAddressModal: '#updateCustomerAddressModal',
-  openOrderAddressUpdateModalBtn: '.js-update-customer-address-modal-btn',
-  updateOrderAddressTypeInput: '#change_order_address_address_type',
-  orderMessageNameSelect: '#order_message_order_message',
-  orderMessagesContainer: '.js-order-messages-container',
-  orderMessage: '#order_message_message',
-  orderMessageChangeWarning: '.js-message-change-warning',
-  allMessagesModal: '#view_all_messages_modal',
-  allMessagesList: '#all-messages-list',
-  openAllMessagesBtn: '.js-open-all-messages-btn',
+  orderPaymentDetailsBtn: ".js-payment-details-btn",
+  orderPaymentFormAmountInput: "#order_payment_amount",
+  orderPaymentInvoiceSelect: "#order_payment_id_invoice",
+  viewOrderPaymentsBlock: "#view_order_payments_block",
+  privateNoteToggleBtn: ".js-private-note-toggle-btn",
+  privateNoteBlock: ".js-private-note-block",
+  privateNoteInput: "#private_note_note",
+  privateNoteSubmitBtn: ".js-private-note-btn",
+  addCartRuleModal: "#addOrderDiscountModal",
+  addCartRuleApplyOnAllInvoicesCheckbox:
+    "#add_order_cart_rule_apply_on_all_invoices",
```

Comment:
Why do we have these weird line breaks?

Reply:
I don't now myself, that's from the original PR

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479